The tests on `jupyter-fs` fail on recent Python versions

See for instance the logs extracted from [this run](https://github.com/mwouts/jupytext/actions/runs/24832470276/job/72684624536):
```
_____________ test_config_jupytext_jupyter_fs_meta_manager[async] ______________
[gw1] linux -- Python 3.13.13 /home/runner/work/jupytext/jupytext/.pixi/envs/default/bin/python3.13

tmpdir = local('/tmp/pytest-of-runner/pytest-0/popen-gw1/test_config_jupytext_jupyter_f0')
cm_from_fs_meta_manager = <jupytext.async_contentsmanager.build_async_jupytext_contents_manager_class.<locals>.AsyncJupytextContentsManager object at 0x7f8aaa16e270>

    @pytest.mark.asyncio
    async def test_config_jupytext_jupyter_fs_meta_manager(tmpdir, cm_from_fs_meta_manager):
        """Test the configuration of Jupytext with a fs manager"""
        tmpdir.join("jupytext.toml").write('formats = "ipynb,py"')
        cm = cm_from_fs_meta_manager
        # the hash that corresponds to the osfs
>       osfs = [h for h in cm._managers if h != ""][0]
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E       IndexError: list index out of range

tests/external/jupyter_fs/test_jupyter_fs.py:83: IndexError
------------------------------ Captured log setup ------------------------------
WARNING  jupyter-fs:metamanager.py:71 Resource {'url': 'osfs:///tmp/pytest-of-runner/pytest-0/popen-gw1/test_config_jupytext_jupyter_f0'} missing 'auth' key, defaulting to 'ask'
WARNING  jupyter-fs:metamanager.py:75 Resource {'url': 'osfs:///tmp/pytest-of-runner/pytest-0/popen-gw1/test_config_jupytext_jupyter_f0', 'auth': 'ask', 'type': 'pyfs'} missing 'type' key, defaulting to 'pyfs'
ERROR    jupyter-fs:metamanager.py:139 Missing dependencies to create manager for resource None
Traceback (most recent call last):
  File "/home/runner/work/jupytext/jupytext/.pixi/envs/default/lib/python3.13/site-packages/jupyterfs/metamanager.py", line 122, in initResource
    managers[_hash] = manager_type.create(
                      ~~~~~~~~~~~~~~~~~~~^
        urlSubbed,
        ^^^^^^^^^^
    ...<5 lines>...
        },
        ^^
    )
    ^
  File "/home/runner/work/jupytext/jupytext/.pixi/envs/default/lib/python3.13/site-packages/jupyterfs/manager/fs.py", line 146, in create
    from fs.errors import FSError
  File "/home/runner/work/jupytext/jupytext/.pixi/envs/default/lib/python3.13/site-packages/fs/__init__.py", line 4, in <module>
    __import__("pkg_resources").declare_namespace(__name__)  # type: ignore
    ~~~~~~~~~~^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'pkg_resources'
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,13 @@
 Jupytext ChangeLog
 ==================
 
+1.19.2.dev0 (unreleased)
+------------------------
+
+**Changed**
+- We have skipped the tests that involve `jupyterfs` on Python 3.12+ as they started failing on the CI with no obvious way to fix them ([#1509](https://github.com/mwouts/jupytext/issues/1509))
+
+
 1.19.1 (2026-01-25)
 -------------------
 

--- a/src/jupytext/version.py
+++ b/src/jupytext/version.py
@@ -1,4 +1,4 @@
 """Jupytext's version number"""
 
 # Must match [N!]N(.N)*[{a|b|rc}N][.postN][.devN], cf. PEP 440
-__version__ = "1.19.1"
+__version__ = "1.19.2.dev0"

--- a/tests/external/jupyter_fs/test_jupyter_fs.py
+++ b/tests/external/jupyter_fs/test_jupyter_fs.py
@@ -1,4 +1,5 @@
 import logging
+import sys
 
 import pytest
 from jupyter_server.utils import ensure_async
@@ -6,6 +7,8 @@ from nbformat.v4.nbbase import new_code_cell, new_markdown_cell, new_notebook
 
 import jupytext
 from jupytext.compare import compare_cells, notebook_model
+
+pytestmark = pytest.mark.skipif(sys.version_info >= (3, 12), reason="https://github.com/mwouts/jupytext/issues/1509")
 
 
 @pytest.fixture(params=["sync", "async"])

